docs/administration-guide/welcome-to-metabase.md

```diff
@@ -0,0 +1,25 @@
+## Welcome to Metabase!
```

Comment:
should this be in the admin guide?

Reply:
Yeah, I initially put this in the admin guide, then decided it was better as an additional to the user guide's "what is metabase" page so I put it in there, but forgot to remove it from the admin guide. I'll remove it.